images/linux/scripts/installers/docker-moby.sh

```diff
@@ -16,16 +16,22 @@ if ! IsInstalled $docker_package; then
     apt-get remove -y moby-engine moby-cli
     apt-get update
     apt-get install -y moby-engine moby-cli
+    apt-get install --no-install-recommends -y moby-buildx
 else
     echo "Docker ($docker_package) is already installed"
 fi
 
 # Run tests to determine that the software installed as expected
 echo "Testing to make sure that script performed as expected, and basic scenarios work"
+echo "checking the docker-moby and moby-buildx"
 if ! command -v docker; then
     echo "docker was not installed"
     exit 1
+elif ! command -v docker buildx build; then
```

Comment:
This comparison contains logical error: we can have docker and missing buildx validation